Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
External is going to be the default lua option with rpm 5.4.16, with the upgrade to lua 5.3.0.
How do you want to handle the other dependencies ? BDB is somewhat on the heavy side...